rest/service/src/v0/jsongrpc.rs

```diff
@@ -0,0 +1,35 @@
+//! Provides a REST interface to interact with JSON gRPC methods.
+//! These methods are typically used to control SPDK directly.
+
+use super::*;
+use mbus_api::v0::JsonGrpcRequest;
+
+/// Configure the functions that this service supports.
+pub(crate) fn configure(cfg: &mut paperclip::actix::web::ServiceConfig) {
+    cfg.service(json_grpc_call);
+}
+
+// A PUT request is required so that method parameters can be passed in the
+// body.
+//
+// # Example
+// To create a malloc bdev:
+// ```
+//  curl -X PUT "https://localhost:8080/v0/jsongrpc/mayastor/bdev_malloc_create" \
+//  -H "accept: application/json" -H "Content-Type: application/json" \
+//  -d '{"block_size": 512, "num_blocks": 64, "name": "Malloc4"}'
+// ```
+#[put("/v0", "/jsongrpc/{node}/{method}", tags(JsonGrpc))]
```

Comment:
I think this be /nodes/{node}/jsonrpc/{method}

Reply:
Done, thanks :+1: